pandas/io/formats/terminal.py

```diff
@@ -99,14 +99,17 @@ def _get_terminal_size_tput():
         proc = subprocess.Popen(["tput", "cols"],
                                 stdin=subprocess.PIPE,
                                 stdout=subprocess.PIPE)
-        output = proc.communicate(input=None)
-        cols = int(output[0])
+        output_cols = proc.communicate(input=None)
         proc = subprocess.Popen(["tput", "lines"],
                                 stdin=subprocess.PIPE,
                                 stdout=subprocess.PIPE)
-        output = proc.communicate(input=None)
-        rows = int(output[0])
-        return (cols, rows)
+        output_rows = proc.communicate(input=None)
+        try:
```

Comment:
This try/ except should go below the catching of the OSError.

Reply:
Very well, I'll just declare the output variables outside of the try/catch block, I suppose

Reply:
@TomAugspurger I've updated it. Is this what you meant?

Reply:
I don't think you need the prior initializations unless you're going to refer, after a try/except block, to a variable that doesn't get created due to an exception. That can't happen here because of the returns, but OTOH the initializations don't hurt anything, and might make the code less likely to break due to a careless change in the future.

I'm any case, thanks for the fix!